.travis.yml

```diff
@@ -12,4 +12,7 @@ rvm:
 env:
   JRUBY_OPTS=--2.0
 
+before_install:
+    - gem install bundler
```

Comment:
We found out that we needed to install the `bundler` gem before attempting to install other dependencies. There were some issues with the JRuby build when we didn't have these two lines.